src/app/core/teams/teams.service.spec.js

```diff
@@ -336,9 +336,8 @@ describe('Team Service:', () => {
 			afterTeamCount.should.equal(beforeTeamCount - 1);
 
 			// Verify team entry is removed from user
-			const uResult = await TeamMember.findById(user.explicit._id);
-			const userTeam = uResult.teams.find((t) => t._id.equals(team.teamWithNoExternalTeam._id));
-			should.not.exist(userTeam);
+			const count = await TeamMember.count({ 'teams._id': team.teamWithNoExternalTeam._id });
```

Comment:
@jrassa minor, but it looks like the call to `teamsService.deleteTeam` in these two tests is still including the user as a second parameter. That should only have the team parameter, right?

Reply:
Fixed along with a few more.